### PR TITLE
Update the FreeBSD test image to 11.4-release

### PR DESCRIPTION
**Pull request by asomers**
And temporarily disable travis targets

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 freebsd_instance:
-  image_family: freebsd-11-3-snap
+  image_family: freebsd-11-4-release
```
Comment (fkorotkov) [inline .cirrus.yml:2]: You either should use `image` for `freebsd-11-4-release` or:

```suggestion
  image_family: freebsd-11-4
```

Comment (asomers) [reply .cirrus.yml:2]: Thanks for the tip, @fkorotkov .

**Comment by asomers** [comment]
@fkorotkov do you know why the build fails?  When trying to start a FreeBSD instance on Cirrus, it fails with the error "Failed to start an instance: INVALID_ARGUMENT: Bad Request".

**Comment by fkorotkov** [comment]
My bad, the image should be `freebsd-11-4-release-amd64`

**Comment by asomers** [comment]
> My bad, the image should be `freebsd-11-4-release-amd64`

Thanks for the tip.  It's annoying that Google doesn't make it easy to discover the correct image name through the web UI.

**Pull request merged**